cpp/ps0003/ps0003_sol.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+int main()
+{
+    // prompting user for number 2/2
+    int number;
+    std::cout << "please enter the number\n";
+    std::cin >> number;
```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number